pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -2700,6 +2700,8 @@ func checkAndSetIoMember(ctx *domainContext, ib *types.IoBundle, isPort bool, pu
 	// For a new PCI device we check if it exists in hardware/kernel
 	long, err := types.IoBundleToPci(ib)
 	if err != nil {
+		log.Warnf("checkAndSetIoMember: can't get PCiLong for ioBundle. err=%s",
```

Comment:
Please include at least ib.Name in the log message.

Reply:
Or even better follow the log message style for this function with "(type name assignmentgroup)"

Reply:
Done.. All logmessage are now using a common prefix..